example.js

```diff
@@ -73,11 +81,26 @@ const setting = new Settings("Amaterasu", "data/settings.json", "data/ColorSchem
 
     // or you send an array (which i personally prefer)
     .addChangelog(changelog)
+    // This is an example of how to sort your categories by the order in the array
+    .setCategorySort((a, b) => categories.indexOf(a.category) - categories.indexOf(b.category))
+    // This is how you set your gui position (in percent)
+    // The params (xCoord, yCoord)
+    .setPos(15, 15)
+    // This is how you set your gui size (in percent)
+    // The params (width, height)
+    .setSize(70, 70)
+    // This is how to apply all your previously chained methods to the gui (Must come at the end of the method chain)
+    .apply()
 
 // This is how to get settings values
 setting.settings.testFeature
 
 // You could also do (if it wasn't obvious enough)
 // in case you have a feature variable check
 const configName = "testFeature"
-setting.settings[configName]
\ No newline at end of file
+setting.settings[configName];
+
+// This is how you translate your Vigilance settings to Amaterasu
+import {convertToJSON} from "./core/VigilanceSettings"
+import oldSettings from "./oldSettingsPath"
+convertToJSON(oldSettings, "moduleName")
```

Comment:
no `;`!
also mind removing the bottom part ? we have a wiki that explains how to use it^^